client.go

```diff
@@ -291,6 +300,10 @@ func (c *TunaSessionClient) getPubAddrs(includePrice bool) *PubAddrs {
 	}
 	addrs := make([]*PubAddr, 0, len(c.tunaExits))
 	for _, tunaExit := range c.tunaExits {
+		if tunaExit == nil {
```

Comment:
This shouldn't happen at all.
We need to make sure all c.tunaexits are non empty

Reply:
Yes, deleted.